core/src/main/java/org/akaza/openclinica/service/crfdata/EnketoUrlService.java

```diff
@@ -248,9 +250,16 @@ public QueriesBean buildQueryElement(ItemData itemdata) {
             }
             query.setComment(dn.getDetailedNotes());
             query.setStatus(dn.getResolutionStatus().getName().toLowerCase());
-            query.setDate_time(dn.getDateCreated().toString());
+
```

Comment:
Enketo expects the following date/timestamp format to work:2017-03-07 17:58:43 -05:00 , otherwise , it will not work.
The date/time format coming from Postgres db is as such:     2017-03-07 17:58:15.068-05
I can't find in java.time any function that will do the trick to convert to the desired format.
 The only one I found was the joda.time (DateTime) function that worked with me. 
I'm running out of options.Thanks